pkgs/by-name/ap/apriltag/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  opencv4,
+  python3,
+}:
+
+let
+  opencv4WithGtk = opencv4.override {
+    enableGtk2 = true; # For GTK2 support
+    enableGtk3 = true; # For GTK3 support
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "AprilTags";
```

```suggestion
  pname = "apriltags";
```
Please review the [package naming guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming); all `pname` attributes must be lowercase.

---

Thanks for the notes! Will apply the suggestions.